README.md

```diff
@@ -16,12 +16,20 @@ Please create a new environment file called .env, and set up the following varia
 
 * FLOWER_USERNAME=username #Flower Moinitor Username
 * FLOWER_PASSWORD=password #Flower Moinitor Password
-* MAX_MEMORY_PER_CHILD=12000 # Max memory can be allocated a task
-* TIME_LIMIT=180 #Time limit per task
+* MAX_MEMORY_PER_DOCKER_WORKER=500M #Max memory(in Kib) each Celery worker/container can consume
+* WORKER_NUMBERS=5 #Number of Celery worker/containers
```

Comment:
Best to default these to the UniMelb limits we'll be using. They match?

Reply:
see https://github.com/AI-Planning/planning-as-a-service/commit/0198b0798be87069928fcaa58a3ed1147d6ad819